### ISSUE TITLE
List view for app

### ISSUE THREAD

**Issue by ricardo777** [owncloud/richdocuments#99]
Hello,

A list view would be great instead of huge icons.

Maybe a list view with details of date and size?

**Comment by mar1u5** [comment]
I dont think we need this, because you can already view your files in the files-app in a list view. What would be the difference?

**Comment by kieranr** [comment]
Time stamp - sort by recent, sort by name.

Both quite useful - and yeah, you can see in "files-app" - but... that's all files, so in my case, jpg/png etc etc... so I'd like to see this.

**Comment by mar1u5** [comment]
Recent File Listing will come for Nextcloud 11 ... https://github.com/nextcloud/server/pull/292
